src/AltinnCore/Common/Services/Implementation/ExecutionSILocalDev.cs

```diff
@@ -103,24 +97,23 @@ public ServiceContext GetServiceContext(string org, string service, string editi
     /// </summary>
     /// <param name="org">The Organization code for the service owner</param>
     /// <param name="service">The service code for the current service</param>
-    /// <param name="edition">The edition code for the current service</param>
     /// <param name="viewName">The view name</param>
     /// <returns>The name of the RazorView</returns>
-    public string GetRazorView(string org, string service, string edition, string viewName)
-    {
-      var views = _viewRepository.GetViews(org, service, edition);
-      var result = views.GetDefaultRazerViewName(viewName);
-      return result;
-    }
+    ////public string GetRazorView(string org, string service, string viewName)
```

Comment:
Why is this just commented out and not removed?

Reply:
No reason, removed it.